README.adoc

```diff
@@ -271,6 +344,47 @@ https://localhost:9443/metrics[https://localhost:9443/metrics^] URL, or access o
 https://localhost:9443/metrics/application[https://localhost:9443/metrics/application^] URL.
 
 You can see the system metrics in the https://localhost:9443/metrics/base[^] URL as well as see the vendor metrics in the https://localhost:9443/metrics/vendor[^] URL.
+endif::[]
+
+// cloud-hosted guide instructions:
+ifdef::cloud-hosted[]
+Run the following curl command to review the all available metrics that have been enabled through MicroProfile Metrics. 
```

Comment:
suggested edit:

Run the following curl command to review all the metrics that are enabled through MicroProfile Metrics.